src/GameStateMatch.cpp

```diff
@@ -34,16 +34,16 @@ bool GameStateMatch::load() {
     } else {
         level.levelTexture.setDimensions(2000, 2000);
```

Comment:
Magic numbers should be avoided, could we make this a const int if the dimensions are used all over like this?